web/index.html

```diff
@@ -488,13 +632,12 @@ <h3>Used in</h3>
                     }
                 });
 
-                // Preload initial chunks
-                for (let i = 0; i < 10; i++){
-                    this.loadChunk(i)
-                }
+                window.addEventListener("scroll", (event) => {
+                    this.scroll = window.pageYOffset;
```

Comment:
This is needed to move the invisible click boxes to switch between panels so that they properly align with the screen since using them as standard sticky elements was not functioning as expected and caused a lot of issues. Doing it using at event listener was the most responsive and accurate way that I found. Computing the value can work sometimes, but sometimes also just breaks or doesn't function as expected.

Reply:
Move to method